src/mavsdk/plugins/telemetry/telemetry_impl.h

```diff
@@ -416,5 +419,12 @@ class TelemetryImpl : public PluginImplBase {
         OffsetStatus gyro_offset;
 
     } _ap_calibration{};
+
+    enum class SysStatusUsed {
```

Comment:
Nitpick: this could be a `SysStatus { Unknown, Used, Unused }` instead of something that sounds like an augmented boolean :grin:

Reply:
Oh yes, good point.